README.md

```diff
@@ -39,30 +37,27 @@ human-readable, machine-parsable error report.
 
 ## Overview
 
-This library generates and `Error` that is specific to Node.js and generally
-more useful than the `Error` that comes with Node.js. This is fine because the
-useful bits of `Error` that comes with Node.js are already non-standard and the
-standard itself is not very useful at all.
+This library generates an `Error` that is more useful than the `Error` that
+comes with Node.js. This is fine because the useful bits of `Error` that comes
```

Comment:
"This is fine…" no longer makes sense because it was addressing the specificity or specialization. Specific might be the wrong word.